amgut/templates/portal.html

```diff
@@ -159,7 +159,7 @@ <h2 style="width:500px;margin-right:0px;float:left;">{% raw tl['MAILING_HEADER_1
                     <form action="{% raw media_locale['SITEBASE'] %}/authed/sample_overview/" method="post" id="sample_overview2_{{ result['barcode'] }}">
                         <input type="hidden" name="barcode" id="barcode" value="{{ result['barcode'] }}"/>
                     </form>
-                    <a href="" onclick="document.forms['sample_overview2_{{ result['barcode'] }}'].submit(); return false;">{{ result['barcode'] }} ({{ escape(result['participant_name']) }})</a>
+                    <a href="" onclick="document.forms['sample_overview2_{{ result['barcode'] }}'].submit(); return false;">{{ result['barcode'] }} ({{ escape(result.get('participant_name', '') or '') }})</a>
```

Comment:
Yes, due to the issue raised in the email thread on the zombies

On Wed, Feb 25, 2015 at 2:33 PM, Yoshiki Vázquez Baeza <
notifications@github.com> wrote:

> In amgut/templates/portal.html
> <https://github.com/biocore/american-gut-web/pull/384#discussion_r25381251>
> :
>
> > @@ -159,7 +159,7 @@ <h2 style="width:500px;margin-right:0px;float:left;">{% raw tl['MAILING_HEADER_1
> >                      <form action="{% raw media_locale['SITEBASE'] %}/authed/sample_overview/" method="post" id="sample_overview2_{{ result['barcode'] }}">
> >                          <input type="hidden" name="barcode" id="barcode" value="{{ result['barcode'] }}"/>
> >                      </form>
> > -                    <a href="" onclick="document.forms['sample_overview2_{{ result['barcode'] }}'].submit(); return false;">{{ result['barcode'] }} ({{ escape(result['participant_name']) }})</a>
> > +                    <a href="" onclick="document.forms['sample_overview2_{{ result['barcode'] }}'].submit(); return false;">{{ result['barcode'] }} ({{ escape(result.get('participant_name', '') or '') }})</a>
>
> Is the reason why you have to do 'or' is because the value of that key
> could be associated with None?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/american-gut-web/pull/384/files#r25381251>.
>